mods/sgssb/moves.js

```diff
@@ -418,12 +418,24 @@ exports.BattleMovedex = {
 		isNonstandard: true,
 		name: "Mewtation",
 		pp: 10,
+		secondary: {
+			chance: 100,
```

Comment:
This is just me nitpicking but if a poke with shield dust came it. The opponent wouldn't get toxic'd